### ISSUE TITLE
Improve documentation

### ISSUE THREAD

**Issue by Xandaros** [3arthqu4ke/headlessmc#215]
This project could really do with some additional documentation.

I just set it up, following the "How to use" section of the Readme. The first thing it did after starting me, is tell me I should be using the launcher wrapper, not the launcher directly. The Readme told me to use the launcher directly, though.
It does not mention this wrapper at all, (except for Android and plugins) nor is it anywhere mentioned what it actually does. All I know is that "some things will not work properly"

The configuration is also confused. It does create a config file, with a single value in it - a list of possible JVMs. What other options are there?
The Readme points directly at the code, which is a start - I at least know what keys exist. No descriptions, though. I wanted to change the game's directory away from `~/.minecraft`, but I honestly still don't know how. There is `hmc.mcdir` and `hmc.gamedir`. What is the difference? No idea. I don't know the default values either, nor do I know of a way to get HMC to tell me the current value.

> Additional configs can be added to the `HeadlessMc/configs` folder.

What does this even mean? Are files in there the same as the config.properties or is there a difference? What's the point?

In summary, here's what I would like to see:
- Update "How to Use" section to include the wrapper
- Explain each config option, either in the Readme, or using the software itself
- Add an option to see the current config, including default values
- Explain what the configs folder is for

All that being said, once I figured things out, it seems to work pretty well. Running it through Xvfb , which initially tried to set my CPU on fire. After tweaking graphics settings, it running really well. Did take me a moment to find a suitable HMC Specifics mod though (1.20.1 forge)
I'm glad to see projects like this still exist, after I abandoned [my own](https://github.com/Xandaros/MinecraftCLI) so long ago :D

**Comment by 3arthqu4ke** [comment]
Good points, I have plans for a large refactoring which will solve all of these issues and more, but sadly I do not really have the time right now.

Its probably pointless since you figured ir out and this should be in the README but here you go:
- the launcher-wrapper message could definitely be worded better, it only affects the plugin system (there are no plugins) and the inmemory launcher (launches the game in the same jvm, only for platforms where we cant create a process).
- `hmc.mcdir` is for libraries and assets, `hmc.gamedir` where the game runs, for mods and world saves etc. by default the same directory.
- configs folder is really not thought out right now, you can place other .properties files there that you can load with a command, that  way you could e.g. have different "profiles" for different versions.
